Anchor.toml

```diff
@@ -27,4 +27,4 @@ cluster = "localnet"
 wallet = "~/.config/solana/id.json"
 
 [scripts]
-test = "yarn run ts-mocha -t 1000000 tests/**/*-test.ts"
+test = "yarn run ts-mocha -t 1000000 tests/**/*-test.ts"
```

Comment:
Configure your editor to add trailing newlines so that we don't get these non-changes in PRs!

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files